src/common/types/inpage-types.ts

```diff
@@ -10,6 +10,7 @@ export enum DomEventName {
   signPsbtRequest = 'SatsPsbtRequest',
   signMessageRequest = 'SatsSignMessage',
   sendBtcRequest = 'SatsSendBtcRequest',
+  createInscriptionRequest = 'createInscriptionRequest',
```

Comment:
why did we not follow the pattern and assign 'SatsCreateInscriptionRequest'

Reply:
Ah, good catch. I didn't notice the pattern. Thanks 😄